auth_supplier/__manifest__.py

```diff
@@ -7,7 +7,7 @@
 {
     'name': "Auth Supplier",
     'category': 'Portal',
-    'version': '9.0.2.0.0',
+    'version': '10.0.1.0.0',
```

Comment:
This is non sense. Odoo releasing a version each year, modules won't have same version despite sharing exact same code.

Also, referring to [this](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers), there is no indication on resetting the module version.

The two first digits are just here to tell that the module can be installed only on a specific odoo version.

For this module, the version should even be `10.0.2.1.0` if you consider adding new translations is a feature.